installer/opensuse/debootstrap.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh -e
+# Copyright (c) 2017 The crouton Authors. All rights reserved.
```

Comment:
You can't claim copyright just by deleting code, and most (all?) of this file appears to be copied from [here](https://github.com/openSUSE/debootstrap/blob/master/debootstrap.sh).

There's also no reason to fake the interface of debootstrap.  Please make an effort to implement opensuse/bootstrap without the use or reference of this code.  I suspect you can write what you need from scratch in a few lines inside opensuse/bootstrap.